src/app/dao/tests/FirebaseFirestoreDAO.spec.ts

```diff
@@ -205,10 +205,12 @@ describe('FirebaseFirestoreDAO', () => {
     describe('subCollectionDAO', () => {
         it('should provide the subcollection with the fully correct path', async() => {
             // Given a dao with a certain path
+            // eslint-disable-next-line dot-notation
```

Comment:
si la CollectionReference est immutable, on peut la rendre public readonly plutôt que private readonly et du coup éviter de disable and break rule ? (vaut pour le suivant naturlich)

Reply:
Fait ici, ça a du sens. En général, c'est quand même mieux de préférer une private pour éviter qu'on l'utilise à tout va.